src/prescriptions/prescription-tab-panel.component.tsx

```diff
@@ -39,13 +39,13 @@ const PrescriptionTabPanel: React.FC<PrescriptionTabPanelProps> = ({ status, isT
         {config.locationBehavior?.locationFilter?.enabled &&
           !isFilterLocationsLoading &&
           filterLocations?.length > 1 && (
-            <ComboBox
+            <MultiSelect
               id="locationFilter"
```

Comment:
```suggestion
              hideLabel
              id="locationFilter"
```

Recommend adding this prop to hide the title text and fix the vertical alignment bug:

<img width="1646" height="142" alt="Image" src="https://github.com/user-attachments/assets/7996b419-0570-4d85-8b46-8d3c00b7a972" />

I also recommend tweaking the .searchContainer class in `prescriptions.scss` so that we replace `flex-direction: row-reverse` with `justify-content: flex-end`. We could also add a gap: `gap: layout.$spacing-05`.

That'd give us:

<img width="2548" height="372" alt="Image" src="https://github.com/user-attachments/assets/fbf8d041-a180-412b-8ca8-28c4f48d1799" />

Reply:
Thanks @denniskigen  I took your advice and tweaks the prescription.scss, it definitely looks better!